Code/GraphMol/ChemTransforms/MolFragmenter.cpp

```diff
@@ -357,26 +356,46 @@ namespace {
 void checkChiralityPostMove(const ROMol &mol, const Atom *oAt, Atom *nAt,
                             const Bond *bond) {
   INT_LIST newOrder;
-  ROMol::OEDGE_ITER beg, end;
-  boost::tie(beg, end) = mol.getAtomBonds(oAt);
-  while (beg != end) {
-    const Bond* obond = mol[*beg];
-    ++beg;
-    if (obond == bond) {
-      continue;
+  INT_LIST incomingOrder;
+  // since we may call this function more than once, we need to keep track of
+  // whether or not we've already been called and what the new atom order is.
+  // we do this with a property.
+  // this was github #1734
+  if (nAt->getPropIfPresent("_newBondOrder", incomingOrder)) {
```

Comment:
We should add "_newBondOrder" to RDGeneral/types.h

Reply:
Or make it a constant.  Should we remove these props at the end?  What happens the next time this is called?

Reply:
They are removed at the end when clearComputedProps() is called

Reply:
I will add it as a constant here (it’s not used elsewhere)